AUTHORS.md

```diff
@@ -155,8 +166,10 @@ possible:
  * Richard Miller
  * Rintcius Blok
  * Rob Norris
+ * Rohan Shah
  * Romain Ruetschi
  * Ross A. Baker
+ * rsoeldner
```

Comment:
Seems rsoeldner is also a couple of lines below in the list

Reply:
thanks good catch